src/graph.mojo

```diff
@@ -0,0 +1,163 @@
+from collections import InlineArray  # , Dict
+from bitmask import BitMask
+
+# We use a stupid dict to circumvent a current
+# bug in the compiler causing a segfault when
+# using the Dict type.
+from supid_dict import StupidDict as Dict
+
+
+@value
+struct Node[DataType: AnyTrivialRegType](CollectionElement):
+    """Node in a BitMaskGraph.
+
+    Parameters:
+        DataType: The type of the value stored in the node.
+    """
+
+    # The index indicating a non-established link.
+    alias null_index = -1
+
+    # The value stored in the node.
+    var value: DataType
+
+    # The indices of the neighbouring nodes.
+    # The node at index i difffers from the
+    # current node by the i-th bit.
+    var neighbours: InlineArray[Int, 256]
```

Comment:
Ah, I think now I understand what you mean by the flip simplification... holding only one list of neighbors instead of one for addition and one for removal. Just logic that this is possible, ut didn't think that far.

Reply:
Used the opportunity to change the implementation in Arche accordingly.